tests/conftest.py

```diff
@@ -107,7 +107,3 @@ def _log_filter(
 @pytest.fixture(scope='session')
 def port_range():
     return glbl_cfg().get(['scheduler', 'run hosts', 'ports'])
-
-
-def pytest_addhooks(pluginmanager):
-    pluginmanager.unregister('tornado')
```

Comment:
This fix seems to have stopped working for me 🤷‍♂️ Replaced by an addition to `pytest.ini`

Reply:
The `-p` approach works for me so fair'nough.